osu.Game/Database/LegacyScoreExporter.cs

```diff
@@ -28,7 +26,9 @@ protected override string GetFilename(ScoreInfo score)
             return filename;
         }
 
-        public override void ExportModelTo(ScoreInfo model, Stream outputStream)
+        protected override string FileExtension => ".osr";
+
+        public override void ExportToStream(ScoreInfo model, Stream outputStream, ProgressNotification? notification, CancellationToken cancellationToken = default)
```

Comment:
This isn't handling the case of missing file in the same way as `LegacyArchiveExporter` - but I'm looking away not to add more stuff onto of this trash fire of a branch than absolutely necessary (and I still want to push in a few *other* touch-ups).